src/components/ResultCard/ResultCard.tsx

```diff
@@ -90,9 +100,7 @@ export default function ResultCard(props: ResultCardOptions) {
 
   return (
     <div className='cio-result-card'>
-      {customAddToFavoritesCallback && (
-        <ResultFavoritesButton item={result} price={salePrice || regularPrice} />
-      )}
+      {customAddToFavoritesCallback && <ResultFavoritesButton item={result} price={salePrice} />}
```

Comment:
@mocca102 It looks like we removed regularPrice from here while testing this out, can we bring it back?